packages/terra-badge/src/_variables.scss

```diff
@@ -1,57 +1,80 @@
-$terra-badge-font-sizes: (
-  tiny: 0.5rem,
-  small: 0.8rem,
-  medium: 1rem,
-  large: 1.4rem,
-  huge: 1.7rem
-);
+:root {
```

Comment:
Looks like this is very easy to do with a PostCSS plugin https://github.com/cbracco/postcss-remove-root

Even if we don't want to use that plugin, the impl of that plugin is very short.